freeradius3: Fix proxy.conf file conflict and enable the rlm_python and rlm_python3 modules

Robby- opened the pull request:
Maintainer: no maintainer at this time
Compile tested: ARM, Linksys EA3500, master
Run tested: No

Description:
- Fix proxy.conf file conflict. This closes #11169.
- Enable the rlm_python and rlm_python3 modules. This closes #9572.

Robby- commented:
Unsure why the `rlm_sql` / `rlm_sql_mongo` failure is now creeping up.

jefferyto commented:
I think it's trying to build other experimental modules; rlm_sql_mongo isn't on the [list of stable rlm_sql drivers](https://github.com/FreeRADIUS/freeradius-server/blob/release_3_0_20/src/modules/rlm_sql/stable).

Rather than trying to prevent other experimental modules from building (which I'm not sure is possible without a lot of effort), I think it would be easier to patch rlm_python3 into the [list of stable modules](https://github.com/FreeRADIUS/freeradius-server/blob/release_3_0_20/src/modules/stable) (so that `--with-experimental-modules` isn't necessary).

Robby- commented:
Fixed, now builds successfully.

I fixed it by removing `--with-experimental-modules --with-rlm_python3` and using `--with-modules="rlm_python3"` instead.
